incl/comments/uploadGJComment.php

```diff
@@ -20,7 +20,7 @@
 $uploadDate = time();
 $decodecomment = base64_decode($comment);
 if(Commands::doCommands($id, $decodecomment, $levelID)){
-	exit("-1");
+	exit("temp_0_Executed command successfully!");
```

Comment:
This is not how the PHP exit() function works in Geometry Dash. It must be an integer, not a string.

Reply:
that is not true, comment bans use this format

Reply:
Really? Then is the `temp_0` part necessary?

Reply:
It is, `temp_0` tells GD that it's a temporary comment ban for 0 seconds and "Executed command succesfully!" will then show up as the ban reason.

Reply:
Well the game is changed the last time I left at. So I see no issues.

Reply:
The PHP exit function just returns a value of any type and quits.